sympy/physics/control/lti.py

```diff
@@ -249,6 +261,108 @@ def expand(self):
+        limit as frequency approaches to zero. For systems with pure
+        integrator terms, DC gain is infinite.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 3, s**2 - 9, s)
+        >>> tf1.dcgain()
+        -1/3
+        >>> tf2 = TransferFunction(p**2, p - 3 + p**3, p)
+        >>> tf2.dcgain()
+        0
+        >>> tf3 = TransferFunction(a*p**2 - b, s + b, s)
+        >>> tf3.dcgain()
+        (a*p**2 - b)/b
+
+        """
+        m = Mul(self.num, Pow(self.den, -1, evaluate=False), evaluate=False)
+        return limit(m, self.var, 0)
+
+    def pole(self):
+        """
+        Returns the poles of a transfer function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction((p + 3)*(p - 1), (p - 1)*(p + 5), p)
+        >>> tf1.pole()
+        [-5, 1]
+        >>> tf2 = TransferFunction((1 - s)**2, (s**2 + 1)**2, s)
+        >>> tf2.pole()
+        [I, I, -I, -I]
+        >>> tf3 = TransferFunction(s**2, a*s + p, s)
+        >>> tf3.pole()
+        [-p/a]
+
+        """
+        chareq = Poly(self.den, self.var)
+        try:
+            return _roots(chareq, self.var)
+        except NotImplementedError:
+            raise ValueError("Could not compute poles of this transfer function.")
+
+    def zero(self):
+        """
+        Returns the zeros of a transfer function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction((p + 3)*(p - 1), (p - 1)*(p + 5), p)
+        >>> tf1.zero()
+        [-3, 1]
+        >>> tf2 = TransferFunction((1 - s)**2, (s**2 + 1)**2, s)
+        >>> tf2.zero()
+        [1, 1]
+        >>> tf3 = TransferFunction(s**2, a*s + p, s)
+        >>> tf3.zero()
+        [0, 0]
+
+        """
+        chareq = Poly(self.num, self.var)
+        try:
+            return _roots(chareq, self.var)
+        except NotImplementedError:
+            raise ValueError("Could not compute zeros of this transfer function.")
+
+    @property
+    def is_stable(self):
+        """
+        Returns True if the transfer function is stable; else False.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.core.symbol import symbols
+        >>> q, r = symbols('q, r', negative=True)
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction((1 - s)**2, (s + 1)**2, s)
+        >>> tf1.is_stable
+        True
+        >>> tf2 = TransferFunction((1 - p)**2, (s**2 + 1)**2, s)
+        >>> tf2.is_stable
+        False
+        >>> tf3 = TransferFunction(4, q*s - r, s)
+        >>> tf3.is_stable
+        False
```

Comment:
An example of `None` would be good.

Reply:
@namannimmo10 Can you add a `None` example here for the users?